test/integration/admin/api/services_controller_test.rb

```diff
@@ -250,4 +250,41 @@ def provider_services
       System::Database.postgres? ? scope.order(:id) : scope
     end
   end
+
+  class PaginationMetadataTest < ActionDispatch::IntegrationTest
+    disable_transactional_fixtures!
+
+    def setup
+      @provider = FactoryBot.create(:provider_account)
+      host! provider.admin_domain
+      FactoryBot.create_list(:simple_service, 600, account: provider)
+    end
+
+    attr_reader :provider
+
+    test 'json' do
+      perform_request(format: :json)
+      pp parsed_response
```

Comment:
what does `pp` do?

Reply:
Just print on the screen, like "puts" but more beautiful 😄 

That's why I called the commit like this:
![image](https://user-images.githubusercontent.com/11318903/100259503-92160700-2f48-11eb-8888-2dd400fb6e29.png)

Because it is not intended to stay like this. I will remove this commit and add a proper generic test.
This was only to visually quickly see what was going on 😂 I also mentioned this in the description of the Jira.